### PR TITLE
chore, refactor, feat(egen): Follows new template, markdown fixes, refactors

### PR DESCRIPTION
**Pull request by krishr2d2**
<br>
This PR includes the following updates to the `notebooks/official/pipelines/rapid_prototyping_bqml_automl.ipynb` notebook:

- budget_milli_node_hours are specified to reduce the training time.
- removes IS_TESTING condition in the cleanup step.
- replaces REGION with LOCATION.
- replaces dsl.Condition (is deprecated) with dsl.If.
- addd a clean up step for the locally generated pipeline package file.
- adds link for Colab enterprise.
- removes future tense (will).
- contracts words (do not, you are etc.).
- image links are no longer accessible and hence are removed
- fixes typos (aboyt, election).
- removes the usage of "we".

<br><br><br>

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

<br>

### CODE REVIEW

File: notebooks/official/pipelines/rapid_prototyping_bqml_automl.ipynb [hunk 1]
```diff
@@ -61,30 +63,27 @@
...
       "cell_type": "markdown",
       "metadata": {
-        "id": "k-82uiXlTjvw"
+        "id": "c6fc06d85572"
       },
       "source": [
         "### Objective\n",
         "\n",
-        "In this tutorial, you learn how to use `Vertex AI Predictions` for rapid prototyping a model.\n",
+        "In this tutorial, you learn how to use Vertex AI pipelines for rapid prototyping a model.\n",
```
Comment (kittyabs) [inline notebooks/official/pipelines/rapid_prototyping_bqml_automl.ipynb:79]: - change: Vertex AI pipelines
    - to: Vertex AI Pipelines (that's the product name)

File: notebooks/official/pipelines/rapid_prototyping_bqml_automl.ipynb [hunk 2]
```diff
@@ -582,11 +534,13 @@
...
-        "- Make sure the GCS bucket and the BigQuery Dataset do not exist. This script may **delete** any existing content.\n",
+        "### Determine project and pipeline variables\n",
+        "\n",
+        "Instructions before you set the variables:\n",
+        "- Make sure that the GCS bucket and the BigQuery dataset don't exist. This notebook may **delete** any existing content.\n",
         "- Your bucket must be on the same region as your Vertex AI resources.\n",
-        "- BQ region can be US or EU;\n",
-        "- Make sure your preferred Vertex AI region is supported [[link]](https://cloud.google.com/vertex-ai/docs/general/locations#americas_1).\n"
+        "- BQ region can be US or EU.\n",
+        "- Make sure your preferred Vertex AI region(LOCATION) is supported. Check from the [list of supported regions](https://cloud.google.com/vertex-ai/docs/general/locations#americas_1).\n"
```
Comment (kittyabs) [inline notebooks/official/pipelines/rapid_prototyping_bqml_automl.ipynb:543]: remove "from":

- Check the [list of supported regions]

File: notebooks/official/pipelines/rapid_prototyping_bqml_automl.ipynb [hunk 3]
```diff
@@ -636,7 +591,7 @@
       "source": [
         "### Downloading the data\n",
         "\n",
-        "The cell below will download the dataset into a CSV file and save it in GCS"
+        "The cell below downloads the dataset into a CSV file and save it in your Cloud Storage bucket."
```
Comment (kittyabs) [inline notebooks/official/pipelines/rapid_prototyping_bqml_automl.ipynb:594]: change "save" to "saves"

File: notebooks/official/pipelines/rapid_prototyping_bqml_automl.ipynb [hunk 4]
```diff
@@ -656,7 +611,11 @@
         "id": "owlPQF1KF8QO"
       },
       "source": [
-        "## Pipeline Components"
+        "## Define the pipeline components\n",
+        "\n",
+        "Before you run the pipeline, define the individual components for your pipeline.\n",
+        "\n",
+        "**Note**: In this section, you define the custom components that are not available in the Vertex AI SDK by default."
```
Comment (kittyabs) [inline notebooks/official/pipelines/rapid_prototyping_bqml_automl.ipynb:618]: change "are not" to "aren't"

File: notebooks/official/pipelines/rapid_prototyping_bqml_automl.ipynb [hunk 5]
```diff
@@ -665,9 +624,9 @@
         "id": "79eaa73a"
       },
       "source": [
-        "### Import to BQ\n",
+        "### Import to BigQuery\n",
         "\n",
-        "This component takes the csv file and imports it to a table in BigQuery. If the dataset does not exist, it will be created. If a table with the same name already exists, it will be deleted and recreated"
+        "Firstly, define a component that loads the csv file and imports it to a BigQuery table. If the dataset doesn't exist, it gets created. If a table already exists with the same name, it gets overwritten."
```
Comment (kittyabs) [inline notebooks/official/pipelines/rapid_prototyping_bqml_automl.ipynb:629]: - change "Firstly" to "First"
- change "gets" to "it's"
     -  If the dataset doesn't exist, it's created
- delete "gets":
    - If a table already exists with the same name, it's overwritten

File: notebooks/official/pipelines/rapid_prototyping_bqml_automl.ipynb [hunk 6]
```diff
@@ -893,13 +849,13 @@
...
+        "Define a component for creating the BQML model. For this demo, you use a simple linear regression model in BQML. However, you can be creative with other model architectures, such as Deep Neural Networks, XGboost, Logistic Regression, etc.\n",
         "\n",
-        "For a full list of models supported by BQML, look here: [End-to-end user journey for each model](https://cloud.google.com/bigquery-ml/docs/reference/standard-sql/bigqueryml-syntax-e2e-journey).\n",
+        "For a full list of models supported by BQML, see [End-to-end user journey for each model](https://cloud.google.com/bigquery-ml/docs/reference/standard-sql/bigqueryml-syntax-e2e-journey).\n",
         "\n",
-        "As pointed out before, BQML and AutoML use different split terminologies, so we do an adaptation of the <i>split_col</i> column directly on the SELECT portion of the CREATE model query:\n",
+        "As pointed out before, BQML and AutoML use different split terminologies, so you do an adaptation of the <i>split_col</i> column directly on the SELECT portion of the CREATE model query:\n",
```
Comment (kittyabs) [inline notebooks/official/pipelines/rapid_prototyping_bqml_automl.ipynb:858]: change "do" to "make"

-  so you make an adaptation of the

File: notebooks/official/pipelines/rapid_prototyping_bqml_automl.ipynb [hunk 7]
```diff
@@ -963,13 +919,12 @@
...
         "\n",
-        "When you do Hyperparameter tuning on the model creation query, the output of the pre-built component [BigqueryEvaluateModelJobOp](https://google-cloud-pipeline-components.readthedocs.io/en/google-cloud-pipeline-components-1.0.0/google_cloud_pipeline_components.experimental.bigquery.html#google_cloud_pipeline_components.experimental.bigquery.BigqueryEvaluateModelJobOp) will be a table with the metrics obtained by BQML when training the model. In your BigQuery console, they look like the image below. We need to access them programmatically so we can compare them to the AutoML model. \n",
+        "When you do hyperparameter tuning with the model creation query, the output of the pre-built component [BigqueryEvaluateModelJobOp](https://google-cloud-pipeline-components.readthedocs.io/en/google-cloud-pipeline-components-1.0.0/google_cloud_pipeline_components.experimental.bigquery.html#google_cloud_pipeline_components.experimental.bigquery.BigqueryEvaluateModelJobOp) is a table with the metrics obtained by BQML when training the model. To compare them with those obtained for AutoML model, you need to access them programmatically.\n",
```
Comment (kittyabs) [inline notebooks/official/pipelines/rapid_prototyping_bqml_automl.ipynb:924]: - change "pre-built"
    - "prebuilt"

File: notebooks/official/pipelines/rapid_prototyping_bqml_automl.ipynb [hunk 8]
```diff
@@ -1013,13 +968,11 @@
...
-        "### Interpret AutoML Model Evaluation\n",
-        "\n",
-        "Similar to BQML, AutoML also generates metrics during its model creation. These can be accessed in the UI, as seen below:\n",
+        "### Interpret AutoML model evaluation\n",
         "\n",
-        "<img src=\"https://storage.googleapis.com/rafacarv-public-bucket-do-not-delete/abalone/automl-evaluate.png\" />\n",
+        "Similar to BQML, AutoML also generates metrics during its model creation that can be accessed from the Google Cloud console.\n",
         "\n",
-        "Since we don't have a pre-built-component to access these metrics programmatically, we can use the Vertex AI GAPIC (Google API Compiler), which auto-generates low-level gRPC interfaces to the service.\n"
+        "Since there isn't a pre-built component to access the AutoML metrics programmatically, you define the below component. The below code uses Vertex AI GAPIC (Google API Compiler) API which auto-generates low-level gRPC interfaces to the specified service.\n"
```
Comment (kittyabs) [inline notebooks/official/pipelines/rapid_prototyping_bqml_automl.ipynb:975]: change "pre-built" to "prebuilt"

File: notebooks/official/pipelines/rapid_prototyping_bqml_automl.ipynb [hunk 9]
```diff
@@ -1085,11 +1038,11 @@
         "id": "7421c559"
       },
       "source": [
-        "### Model Selection\n",
+        "### Model selection\n",
         "\n",
-        "Now that we have evaluated the models independently, we are going to move forward with only one of them. This election will be done based on the model evaluation metrics gathered in the previous steps.\n",
+        "After the models are evaluated independently, you're going to move forward with only one of them. The selection is done based on the model evaluation metrics gathered in the previous steps. The selected model is then deployed to an endpoint.\n",
```
Comment (kittyabs) [inline notebooks/official/pipelines/rapid_prototyping_bqml_automl.ipynb:1043]: - change: "After the models are evaluated independently, you're going to move forward with only one of them"
    - "After the models are evaluated independently, you're going to only move forward with one of them."

File: notebooks/official/pipelines/rapid_prototyping_bqml_automl.ipynb [hunk 10]
```diff
@@ -1085,11 +1038,11 @@
...
         "\n",
-        "Bear in mind that BQML and AutoML use different evaluation metric names, hence we had to do a mapping of these different nomenclatures."
+        "Define a component to select the best out of the two models that is suitable for deployment. Note that BQML and AutoML use different evaluation metric names, hence you need to do a mapping of these different nomenclatures."
```
Comment (kittyabs) [inline notebooks/official/pipelines/rapid_prototyping_bqml_automl.ipynb:1045]: change "hence" to "therefore"

File: notebooks/official/pipelines/rapid_prototyping_bqml_automl.ipynb [hunk 11]
```diff
@@ -1195,9 +1148,9 @@
         "id": "0f573556"
       },
       "source": [
-        "### Validate Infrastructure\n",
+        "### Validate the infrastructure\n",
         "\n",
-        "Once the best model has been deployed, you validate the endpoint by making a simple prediction to it."
+        "Post selecting the best model, it is deployed to an endpoint. Define a component that validates the endpoint by making prediction requests to it."
```
Comment (kittyabs) [inline notebooks/official/pipelines/rapid_prototyping_bqml_automl.ipynb:1153]: - change "it is" to "it's"
- change: " Define a component that validates the endpoint by making prediction requests to it."
    -to: " Define a component that validates the endpoint by making prediction requests to that endpoint."

File: notebooks/official/pipelines/rapid_prototyping_bqml_automl.ipynb [hunk 12]
```diff
@@ -1275,28 +1228,9 @@
...
-        "    \"bq_dataset\": BQ_DATASET,\n",
-        "    \"bq_location\": BQ_LOCATION,\n",
-        "    \"bqml_model_export_location\": BQML_EXPORT_LOCATION,\n",
-        "    \"bqml_serving_container_image_uri\": BQML_SERVING_CONTAINER_IMAGE_URI,\n",
-        "    \"endpoint_display_name\": ENDPOINT_DISPLAY_NAME,\n",
-        "    \"thresholds_dict_str\": '{\"rmse\": 2.5}',\n",
-        "}"
+        "## Define the pipeline\n",
+        "\n",
+        "Now, define the flow of your pipeline using the pre-built components and the custom components you defined above."
```
Comment (kittyabs) [inline notebooks/official/pipelines/rapid_prototyping_bqml_automl.ipynb:1233]: change "pre-built" to "prebuilt"

**Review by kittyabs — COMMENTED** [review]
numerous edits

**Review by katiemn — APPROVED** [review]

**Pull request merged**